lib/pbench/server/api/resources/upload_api.py

```diff
@@ -51,69 +51,76 @@ def __init__(self, config, logger):
 
     @Auth.token_auth.login_required()
     def put(self, controller):
-
         try:
             username = Auth.token_auth.current_user().username
         except Exception:
             self.logger.exception("Tarfile upload: Exception verifying the username")
             abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
 
-        if not request.headers.get("filename"):
-            self.logger.debug(
-                "Tarfile upload: Post operation failed due to missing filename header"
+        filename_hdr = request.headers.get("filename")
+        if not filename_hdr:
+            self.logger.info(
+                "Tarfile upload: Post operation failed due to missing"
```

Comment:
Not your bug, but it's `PUT` not `POST`!

Reply:
Fixed.